package/drbd.firewalld.xml

```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <service>
   <short>SUSE YaST DRBD</short>
-  <description>This allows you to open various ports related to SUSE YaST DRBD. Port 7788 and higher is recommend.</description>
+  <description>This allows you to open various ports related to SUSE YaST DRBD including Linstor support. 3366/3367 for lnstor satellite. 3370/3376/3377 for linstor controller. 3371 for linstor-controller when using HTTPS REST-API. Port 7788 is the default port of DRBD connection. 7788 and higher are recommend to resource connections.</description>
```

Comment:
Just a small improvement of the text, plus added some new lines to make it better readable in editors.
(I hope new lines are allowed there...)

```suggestion
  <description>This allows you to open various ports related to SUSE
  YaST DRBD including Linstor support. Ports 3366/3367 ares for Linstor
  satellite, ports 3370/3376/3377 are for Linstor controller and
  port 3371 is for Linstor controller when using HTTPS REST-API.
  Port 7788 is the default port for DRBD connection.
  Ports 7788 and higher are recommend for resource connections.</description>```